schemas/com.snowplowanalytics.oss/oss_context/jsonschema/1-0-0

```diff
@@ -0,0 +1,59 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Context schema for OSS terraform modules, VM instances and application events",
+  "self": {
+    "vendor": "com.snowplowanalytics.oss",
+    "name": "oss_context",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
```

Comment:
This schema is not setting any required properties - seems to be something we do on all other schemas?